src/CoreWCF.Http/src/CoreWCF/HttpBindingBase.cs

```diff
@@ -136,9 +137,20 @@ internal TransportBindingElement GetTransport()
             Fx.Assert(BasicHttpSecurity != null, "this.BasicHttpSecurity should not return null from a derived class.");
 
             BasicHttpSecurity basicHttpSecurity = BasicHttpSecurity;
+
             if (basicHttpSecurity.Mode == BasicHttpSecurityMode.Message)
             {
-                throw new PlatformNotSupportedException(nameof(BasicHttpSecurityMode.TransportWithMessageCredential));
+                throw new PlatformNotSupportedException(nameof(BasicHttpSecurityMode.Message));
+            }
+            else if (basicHttpSecurity.Mode == BasicHttpSecurityMode.None && basicHttpSecurity.Transport.ClientCredentialType == HttpClientCredentialType.InheritedFromHost)
+            {
```

Comment:
What's the purpose of this change? I have a suspicion the scenario here that you were looking to affect was with BasicHttpSecurityMode.TransportCredentialOnly

Reply:
Indeed, i reverted back this file and change the unit test configuration. Please see https://github.com/CoreWCF/CoreWCF/pull/850/commits/fc4c36ce39e3a40ae5fac57bdac6f308903ef775

Reply:
You did make one correct fix which was the nameof is the PlatformNotSupportedException is using the wrong enum value. Could you refix that?

Reply:
done here https://github.com/CoreWCF/CoreWCF/pull/850/commits/f2721caf8a5829ea7d741e151452afcb71501fdc